x/gov/test_common.go

```diff
@@ -35,14 +36,18 @@ func getMockApp(t *testing.T, numGenAccs int, genState GenesisState, genAccs []a
 
 	staking.RegisterCodec(mApp.Cdc)
 	RegisterCodec(mApp.Cdc)
+	params.RegisterCodec(mApp.Cdc)
 
 	keyStaking := sdk.NewKVStoreKey(staking.StoreKey)
 	tKeyStaking := sdk.NewTransientStoreKey(staking.TStoreKey)
 	keyGov := sdk.NewKVStoreKey(StoreKey)
 
-	rtr := NewRouter().AddRoute(RouterKey, ProposalHandler)
-
 	pk := mApp.ParamsKeeper
+
+	rtr := NewRouter().
```

Comment:
Imho, this test should live in `x/params/proposal_handler_test.go`. I wanted to avoid having the dependency on `x/params` here.

Reply:
is this resolved ?